pandas/tests/frame/test_timestamps.py

```diff
@@ -0,0 +1,50 @@
+""" test DataFrame-boxed versions of the scalar Timestamp """
+from datetime import datetime
+
+import numpy as np
+
+import pandas.util.testing as tm
+from pandas.tseries import offsets
+
+from pandas.compat import lrange
+from pandas import (Timestamp, date_range,
+                    Series, DataFrame, DatetimeIndex)
+
+
+class TestDataFrameTimestamps(object):
+    def test_series_map_box_timestamps(self):
+        # GH#2689, GH#2627
+        s = Series(date_range('1/1/2000', periods=10))
+
+        def f(x):
+            return (x.hour, x.day, x.month)
```

Comment:
this should be in series

Reply:
It relies on DataFrame (although the whole thing is a smoke test).  Maybe rename test_series_map_box_timestamps--> test_map_box_timestamps?